pkg/api/http_server.go

```diff
@@ -659,6 +664,61 @@ func (hs *HTTPServer) apiHealthHandler(ctx *web.Context) {
 	}
 }
 
+// setupFrontendLogHandlers will set up handlers for logs incoming from frontend.
+// handlers are setup even if frontend logging is disabled, but in this case do nothing
+// this is to avoid reporting errors in case config was changes but there are browser
+// sessions still open with older config
+func (hs *HTTPServer) setupFrontendLogHandlers() {
+	if !(hs.Cfg.GrafanaJavascriptAgent.Enabled || hs.Cfg.Sentry.Enabled) {
+		hs.web.Use(func(ctx *web.Context) {
+			if ctx.Req.Method == http.MethodPost && (ctx.Req.URL.Path == "/log" || ctx.Req.URL.Path == "/log-grafana-javascript-agent") {
```

Comment:
Nit. Consider const for these two paths since referenced here and further down.

Reply:
- [ ] export path literals ot const